parts/k8s/cloud-init/masternodecustomdata.yml

```diff
@@ -104,6 +104,24 @@ write_files:
   content: !!binary |
     {{CloudInitData "aptPreferences"}}
 
+{{if not .MasterProfile.IsCoreOS}}
```

Comment:
you don't need this since `IsIPv6DualStackFeatureEnabled` can only be true if distro != CoreOS